src/types/venues.ts

```diff
@@ -181,6 +181,7 @@ export interface BaseVenue {
   showRangers?: boolean;
   chatTitle?: string;
   showReactions?: boolean;
+  enableJukebox?: boolean;
```

Comment:
can we extend `JazzBarVenue` (not `BaseVenue`) instead?

Reply:
done d04b6de